pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -375,7 +242,7 @@ class Build {
                         context.string(name: 'PRODUCT_MAJOR_VERSION', value: "${versionData.major}"),
                         context.string(name: 'PRODUCT_MINOR_VERSION', value: "${versionData.minor}"),
                         context.string(name: 'PRODUCT_MAINTENANCE_VERSION', value: "${versionData.security}"),
-                        context.string(name: 'PRODUCT_PATCH_VERSION', value: "${versionData.build}"),
+                        context.string(name: 'PRODUCT_PATCH_VERSION', value: "${String.format("%02d", versionData.build)}"),
```

Comment:
Could we ever have 3 decimals?

Reply:
So this is a question re what the installer should expect to see here. For padding build numbers seem to get padded by 1 0, i.e `08`. 

However from what I understand of the installers they are using this info to build up what the directory inside the archive is. But under this new scheme the dir inside the archive is now `jdk-<semver>` so not padded at all. I doubt padded or not will fix this. This is a question for @gdams as to what should be provided here

Reply:
leave it without for now. If your fix is changing the directory name to be the semver version we should be good